index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+   <section>
+       <div class="loader">
+           <span style="--i:1;"></span>

```

Comment:
could you explain what is this `style="--i:1;"` doing here ?

Reply:
we are defining css veriable " i " here and giving it value 1.